3.Tests/UI/StepDefinitions/User/LogoutSteps.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OpenQA.Selenium;
+using SeleniumTest.Core.Drivers;
+using SeleniumTest.Core.Interfaces;
+using SeleniumTest.Tests.Steps.Commons;
+using TechTalk.SpecFlow;
+using Views.WebAppPages;
+
+namespace SeleniumTest.Tests.Steps.User;
+
+[Binding]
+[Scope(Feature = "User Logout")]
+public class LogoutStepDefinitions : CommonSteps
+{
+    private readonly HomePage _homePage;
+    private readonly IGenericWebDriver _driver;
+    private readonly ScenarioContext _scenarioContext;
+
+    public LogoutStepDefinitions(ScenarioContext scenarioContext, ChromeWebDriver driver)
+        : base(scenarioContext, driver)
+    {
+        _scenarioContext = scenarioContext;
+        _driver = driver;
+        _homePage = new HomePage(_driver);
+    }
+
+    [TearDown]
+    public void TearDown()
+    {
+        _driver.Dispose();
+    }
+
+    [When(@"the user clicks the logout button")]
+    public void Whentheuserclicksthelogoutbutton()
+    {
+        _homePage.LogoutOffTheApplication();
+    }
+
+    [Then(@"the user should be logged out from the site")]
+    public void Thentheusershouldbeloggedoutfromthesite()
+    {
+        var div = _driver.Instance().FindElement(By.Id("ctl00_MainContent_PanelNotAuth"));
```

Comment:
What is that div exactly? we should avoid using the webdriver directly on the test layer of our framework, we have to leave that responsibility to the Pages layer, and actually the UIElements layer

Reply:
I guess that div is on the login page, so let the LoginPage class handle this element